netfilter_openvpn.py

```diff
@@ -42,8 +42,7 @@
 import sys
 import ldap
 import fcntl
-import time
-import signal, errno
```

Comment:
errno is used: https://github.com/hu-berlin-cms/openvpn-netfilter/blob/ed441c912f4b5cf9f2d84f5c11ff7e0e2294ef96/netfilter_openvpn.py#L98